src/view/actions/updateVariable.jsx

```diff
@@ -82,6 +86,19 @@ const getInitialFormStateFromDataElement = async ({
       });
     }
   }
+  if (dataElement?.settings?.data) {
+    const schema = generateSchemaFromSolutions(dataElement.settings.data);
```

Comment:
From looking at the manifest, this should be "dataElement.settings.solutions".

Reply:
Good catch.